packages/testing/src/api/index.ts

```diff
@@ -15,10 +15,18 @@
  */
 
 import { firebase } from '@firebase/app';
-import request from 'request-promise';
+import * as request from 'request';
 import { FirebaseApp, FirebaseOptions } from '@firebase/app-types';
 import { base64 } from '@firebase/util';
 import { setLogLevel, LogLevel } from '@firebase/logger';
+import * as grpc from 'grpc';
+import * as fs from 'fs';
+
+const PROTOS = grpc.load({
+  root: __dirname + '/../protos',
```

Comment:
what should this be? can you either get rid of the relative path or have a comment explaining what it is

Reply:
I'm following the existing setup in `packages/firestore`, which does something similar. This is basically the cleanest way of "hardcoding" the path to `src/protos/`